app/main.py

```diff
@@ -1,3 +1,13 @@
 def cache(func):
-    # Write your code here
-    pass
+    saved = dict()
+
+    def function(*args):
+        key = str(args)
```

Comment:
it's not correct, because if in args saved "True" it's a "True", and if saved True it's a "True"
True != "True"